### PR TITLE
adjust string to use default machine name instead of helper path in helper message

### LINKED ISSUE #20579: Helper text prints wrong message
https://github.com/containers/podman/blob/0904c19422934605364b6cbe991b86c12fb6007b/pkg/machine/machine_common.go#L134

The code is (probably) supposed to inform me about a command to stop and then start a machine after I installed podman-helper. However, it's printing the path to the helper instead of the machine name:

```
$ podman machine start
Starting machine "podman-machine-default"
Waiting for VM ...
Mounting volume... /Users:/Users
Mounting volume... /private:/private
Mounting volume... /var/folders:/var/folders

This machine is currently configured in rootless mode. If your containers
require root permissions (e.g. ports < 1024), or if you run into compatibility
issues with non-podman clients, you can switch using the following command:

        podman machine set --rootful

API forwarding listening on: /Users/a1dbe91/.local/share/containers/podman/machine/qemu/podman.sock

The system helper service is not installed; the default Docker API socket
address can't be used by podman. If you would like to install it run the\nfollowing commands:

        sudo /opt/homebrew/Cellar/podman/4.7.2/bin/podman-mac-helper install
        podman machine stop/opt/homebrew/Cellar/podman/4.7.2/bin/podman-mac-helper; podman machine start/opt/homebrew/Cellar/podman/4.7.2/bin/podman-mac-helper

                You can still connect Docker API clients by setting DOCKER_HOST using the
following command in your terminal session:

        export DOCKER_HOST='unix:///Users/a1dbe91/.local/share/containers/podman/machine/qemu/podman.sock'

Machine "podman-machine-default" started successfully
```

Env: 

```
$ podman version
Client:       Podman Engine
Version:      4.7.2
API Version:  4.7.2
Go Version:   go1.21.3
Git Commit:   750b4c3a7c31f6573350f0b3f1b787f26e0fe1e3
Built:        Tue Oct 31 12:59:17 2023
OS/Arch:      darwin/arm64

Server:       Podman Engine
Version:      4.7.0
API Version:  4.7.0
Go Version:   go1.20.8
Built:        Wed Sep 27 20:24:25 2023
OS/Arch:      linux/arm64
```

On an M2Max 16" MacBook Pro

### PR DESCRIPTION
**Pull request by RiRa12621**
#### Does this PR introduce a user-facing change?

this fixes #20579 

```release-note
fixed a helper message to display the right command to execute
```

### CODE REVIEW

**Review by Luap99 — COMMENTED** [review]
Thanks for the contribution, in order to pass CI checks you need to have a shorter commit title.
You can edit the message with `git commit --amend`, you can just remove the `pkg/machine/machine_common` it is not necessary to have this in the title.

**Comment by RiRa12621** [comment]
Thanks, done.

**Comment by ashley-cui** [comment]
Thanks! I think you need to add [NO NEW TESTS NEEDED]

**Comment by ashley-cui** [comment]
Oops misclick, did not mean to close the PR.

**Comment by RiRa12621** [comment]
yes, was just inspecting the pipeline that failed and added it now

**Comment by ashley-cui** [comment]
LGTM, thanks!

File: pkg/machine/machine_common.go [hunk 1]
```diff
@@ -131,7 +131,7 @@ address can't be used by podman. `
 					fmtString += `If you would like to install it, run the following commands:
 
         sudo %s install
-        podman machine stop%[1]s; podman machine start%[1]s
+        podman machine stop%[2]s; podman machine start%[2]s
```
Comment (TomSweeneyRedHat) [inline pkg/machine/machine_common.go:134]: there are a few other `[1]s` instances above this line within this function.  Do they need tending to also?

Comment (RiRa12621) [reply pkg/machine/machine_common.go:134]: probably yes, but i need to verify that

Comment (RiRa12621) [reply pkg/machine/machine_common.go:134]: yes, fixed

**Comment by n1hility** [comment]
There is a lint test failure in the results there that looks legit (not all of the print statements are passed two args)

**Comment by RiRa12621** [comment]
indeed. 
after https://github.com/containers/podman/pull/20580#discussion_r1381886991 i didn't look properly.

the string that's used where the original problem occured is used by `fmt.Printf(fmtString, helper, suffix)` so string and two args and the format. the other string is (where the linter fails now) is actually used by *F*printf --> `fmt.Fprintf(os.Stderr, fmtString, suffix)`. Adjusting now.

**Review by Luap99 — APPROVED** [review]
/lgtm

**Pull request merged**